src/cloud-api-adaptor/Makefile

```diff
@@ -176,6 +176,7 @@ ifeq ($(PODVM_DISTRO),rhel)
 	--build-arg YQ_VERSION=$(YQ_VERSION) \
 	--build-arg YQ_CHECKSUM=${YQ_CHECKSUM_$(ARCH)} \
 	--build-arg YQ_ARCH=$(ARCH) \
+	--build-arg PROTOC_ARCH=$(or $(PROTOC_ARCH),x86_64) \
```

Comment:
Looks like this should be squashed with the previous one - "podvm: adapt RHEL to ARCH=amd64"